### PR TITLE
Add get ready post, and update schedule

### PR DESCRIPTION
**Pull request by ericholscher**
This adds the meetup organizer session & sponsored workshop to the schedule

### CODE REVIEW

**Review by thatdocslady — APPROVED** [review]
Didn't do a deep-dive nit-picky review but at a high level looks great!

File: docs/_data/portland-2022-schedule.yaml [hunk 1]
```diff
@@ -82,17 +82,19 @@ talks_day2:
...
     title: 'Coffee break'
   - duration: '0:30'
     slug: don-t-trust-the-numbers-kumar-dhanagopal
   - duration: '0:05'
     title: '5 minute break'
-  - duration: '0:30'
-    slug: inclusive-language-victor-bracht
+  - duration: '0:25'
+    title: "<b>Meetup information session - Get involved locally!</b>"
+  - duration: '0:05'
```
Comment (rosewms) [inline docs/_data/portland-2022-schedule.yaml:94]: can we do 10 minutes so we can talk about Quorum?

Comment (ericholscher) [reply docs/_data/portland-2022-schedule.yaml:94]: It's currently scheduled for 25 mins (The duration is above it)

Comment (ericholscher) [reply docs/_data/portland-2022-schedule.yaml:94]: It's currently scheduled for 25 mins (The duration is above it)

**Pull request merged**